code/obj/item/device/pda2/base_os.dm

```diff
@@ -755,6 +835,12 @@
 					var/senderassignment = signal.data["sender_assignment"]
 					var/file_ext = signal.data["file_ext"]
 					var/filesize = signal.data["file_size"]
+					var/signalTag = signal.data["tag"]
+					var/signalGroup = signal.data["group"]
+
+					if(signalGroup) //Check to see if we have this ~mailgroup~
```

Comment:
Doesn't this clash with #3178 which allows for groups to be a list?

Reply:
Not anymore it doesnt!